src/components/FeaturePanel/FeaturePanelInDrawer.tsx

```diff
@@ -3,17 +3,26 @@ import { FeaturePanel } from './FeaturePanel';
 import { Drawer } from '../utils/Drawer';
 import {
   DRAWER_PREVIEW_HEIGHT,
+  DRAWER_PREVIEW_PADDING,
   DRAWER_TOP_OFFSET,
 } from '../utils/MobilePageDrawer';
 
 const DRAWER_CLASSNAME = 'featurePanelInDrawer';
 
-export const FeaturePanelInDrawer = () => (
-  <Drawer
-    topOffset={DRAWER_TOP_OFFSET}
-    className={DRAWER_CLASSNAME}
-    collapsedHeight={DRAWER_PREVIEW_HEIGHT}
-  >
-    <FeaturePanel />
-  </Drawer>
-);
+export const FeaturePanelInDrawer = () => {
+  const [height, setHeight] = React.useState(DRAWER_PREVIEW_HEIGHT);
+  return (
+    <Drawer
+      key={`drawer-${height}px`}
+      topOffset={DRAWER_TOP_OFFSET}
+      className={DRAWER_CLASSNAME}
+      collapsedHeight={height}
+    >
+      <FeaturePanel
+        onHeadingHeightChange={(height) => {
+          setHeight(height + DRAWER_PREVIEW_PADDING);
+        }}
```

Comment:
Could you please wrap this callback to `useCallback`? The prop `onHeadingHeightChange` goes to a dependency array of a `useEffect` inside the `FeaturePanel`, and that it's mandatory to have a static reference on each rerender. (Otherwise useEffect will have to umout/mount on each)